qa/workunits/cephtool/test.sh

```diff
@@ -1846,6 +1846,12 @@ function test_mon_cephdf_commands()
   expect_false test $cal_raw_used_size != $raw_used_size
 }
 
+function test_mon_stdin_stdout()
```

Comment:
i am not sure if the backporter should (or is allowed to) add his/her own `Signed-off-by` line in the backport commit. but considering the amount of original work i don't think this is the case.

@smithfarm what do you think?